htslib/klist.h

```diff
@@ -88,9 +88,9 @@
 		size_t size;													\
 	} kl_##name##_t;													\
 	SCOPE kl_##name##_t *kl_init_##name(void) {							\
-		kl_##name##_t *kl = calloc(1, sizeof(kl_##name##_t));			\
+		kl_##name##_t *kl = (kl_##name##_t *)calloc(1, sizeof(kl_##name##_t));			\
 		kl->mp = kmp_init(name);										\
-		kl->head = kl->tail = kmp_alloc(name, kl->mp);					\
+		kl->head = kl->tail = kmp_alloc(name, kl->mp);		\
```

Comment:
Was touching this line an accident? OTOH the lines with the new casts could benefit from adjusting their backslashes.

Reply:
Sort of.  I started updating the backslashes to be neat and then just realised the true scale of the horror so backed out of it, mostly...  I spotted it after submitting the PR but thought it's largely irrelevant given how messy the rest are!  I can undo that line too, or add another separate commit that fixes the hideous whitespace in this file.

It's a bit problematic as it can make comparing upstream trickier, but `-w` should suffice.

Reply:
Apart from the newly lengthened lines, the file lines up nicely with 4-space hard tabs. (Which is an abomination in itself, but at least it's upstream's conventional abomination…)

Reply:
Ugh!  It never dawned on me it'd look OK with a non-standard tab width!

Most of the changed lines are now too long, but the one that isn't I've got indenting the same as the others (when using said abomination).  Also removed the accidental white space change.

Thanks, even though I now feel slightly ill.

Reply:
Turns out Heng's down with the kids 😛